assets/src/scripts/components/hydrograph/domain.js

```diff
@@ -90,6 +90,54 @@ export const getYDomain = function (pointArrays, currentVarParmCd) {
 };
 
 
+/**
+ * Function finds highest negative value (or lowest positive value) in array of tick values, then returns that number's
+ * absolute value
+ * @param {array} tickValues, the list of y-axis tick values
+ * @returns {number} lowestAbsoluteValueOfTicks, the lowest absolute value of the tick values
+ */
+export const getLowestAbsoluteValueOfTickValues = function(tickValues) {
+    let lowestAbsoluteValueOfTicks;
+
+    // if tickValues have negative numbers, pull them out and test them to find the (largest negative) smallest absolute value
+    if (tickValues.some(value => value < 0)) {
+        let negativeTickValues = tickValues.filter(value => value < 0);
+        let highestNegativeValueOfTicks = Math.max(...negativeTickValues);
+        lowestAbsoluteValueOfTicks = Math.abs(highestNegativeValueOfTicks);
+    } else {
+        lowestAbsoluteValueOfTicks = Math.min(...tickValues);
+    }
+
+    return lowestAbsoluteValueOfTicks;
+};
+
+
+/**
+ * Function creates a new set of tick values that will fill in gaps in log scale ticks, then combines this new set with the
+ * original set of tick marks.
+ * @param {array} tickValues, the list of y-axis tick values
+ * @returns {array} fullArrayOfTickMarks, the new full array of tick marks for log scales
+ */
+export const getArrayOfAdditionalTickMarks = function(tickValues) {
+    let additionalTickValues = [];
+    let lowestTickValueOfLogScale = getLowestAbsoluteValueOfTickValues(tickValues);
+
+    // Make a set of tick values that when graphed will have equal spacing on the y axis
+    while (lowestTickValueOfLogScale > 2) {
+        lowestTickValueOfLogScale = Math.ceil(lowestTickValueOfLogScale / 2);
```

Comment:
Would it be possible to make this value a power of 10, or maybe multiple of 5?

Reply:
Sure. I showed the output to Jim and he thought we could go with the multiples of 5 and see how people liked that, so a tweaked the code a bit to make that that happen.